src/api/eventTypes.js

```diff
@@ -130,21 +130,35 @@ type StreamListEvent = $ReadOnly<{|
   streams: $ReadOnlyArray<Stream>,
 |}>;
 
+type StreamUpdateEventBase = $ReadOnly<{|
+  ...EventCommon,
+  type: typeof EventTypes.stream,
+  op: 'update',
+  stream_id: number,
+  name: string,
+|}>;
+
+export type StreamUpdateEvent =
```

Comment:
See https://zulip.com/api/get-events#stream-update for more context.

Yeah -- let's in fact have this link in a comment on this type. See:
https://github.com/zulip/zulip-mobile/blob/main/docs/style.md#api-doc-link

(We don't currently do that on all the existing types, but should do so when making changes.)